crates/theme/src/settings.rs

```diff
@@ -52,13 +109,67 @@ pub struct ThemeSettingsContent {
     #[serde(default)]
     pub theme: Option<String>,
 
+    /// Set the themes to use for light and dark mode. Note that this
+    /// Supersedes the `theme` setting.
+    #[serde(default)]
+    pub themes: Option<ThemeFeatures>,
```

Comment:
I think it would be nice if we could have these settings live under the same `theme` key, since they're mutually-exclusive with it.

This should be possible by introducing a new enum to use as the value for `theme`:

```
#[serde(untagged)]
enum ThemeNameOrDynamic {
    Theme(String),
    Dynamic(ThemeFeatures)
}
```